src/plugins/dataflow/components/dataflow-tile.tsx

```diff
@@ -22,7 +23,9 @@ interface IProps extends ITileProps{
 }
 
 interface IDataflowTileState {
-  programRecordingMode: number // TO DO: convert to enum;
+  programRecordingMode: number; // TO DO: convert to enum
```

Comment:
A convention is to use `TODO:` instead of `TO DO:` this consistency makes it easier to search for these in the code.

Reply:
Got it, ill change and then merge.